Source/Scene/ClippingPlanesCollection.js

```diff
@@ -155,14 +163,20 @@ define([
      * Duplicates this ClippingPlanesCollection instance.
      *
      * @param {ClippingPlanesCollection} [result] The object onto which to store the result.
-     * @returns he modified result parameter or a new ClippingPlanesCollection instance if one was not provided.
+     * @returns {ClippingPlanesCollection} The modified result parameter or a new ClippingPlanesCollection instance if one was not provided.
      */
     ClippingPlanesCollection.prototype.clone = function(result) {
         if (!defined(result)) {
             result = new ClippingPlanesCollection();
         }
 
-        result.planes = Array.from(this.planes);
+        var length = this.planes.length;
+        result.planes = new Array(length);
+        for (var i = 0; i < length; ++i) {
+            var plane = this.planes[i];
+            result.planes[i] = new Plane(plane.normal, plane.distance);
+        }
+
```

Comment:
I didn't notice this before - since `clone` is called every frame it should do its best to avoid allocating a new array and new planes since result will usually already have planes.